test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -14,11 +15,26 @@ public void isAnyNull() {
         // empty list
         assertFalse(Utils.isAnyNull());
 
-        // Any non-empty list
-        assertFalse(Utils.isAnyNull(new Object(), new Object()));
+        // non empty list with one non-null argument
+        assertFalse(Utils.isAnyNull(new ArrayList<Object>()));
+        assertFalse(Utils.isAnyNull(new Object()));
         assertFalse(Utils.isAnyNull("test"));
+        assertFalse(Utils.isAnyNull(1));
         assertFalse(Utils.isAnyNull(""));
-
+        
+        // non empty list with more than one non-null arguments
+        assertFalse(Utils.isAnyNull(new Object(), new Object()));
+        assertFalse(Utils.isAnyNull("item", "item", "item"));
+        assertFalse(Utils.isAnyNull(1,2,3,4,5,6,7,8,9,10));
+        assertFalse(Utils.isAnyNull(" ", "\n", "", 23, "string", new Object()));
+        
+        // non empty list with same objects as arguments
+        Object a = new Object();
+        Object b = a;
+        assertFalse(Utils.isAnyNull(a,b));
+        a = null;
+        assertFalse(Utils.isAnyNull(b));
```

Comment:
This `line 36` is similar to to `line 20` of your test, there is an overlap in the test.